python/ray/serve/_private/router.py

```diff
@@ -296,18 +300,22 @@ def __init__(
         self,
         event_loop: asyncio.AbstractEventLoop,
         deployment_name: str,
+        self_node_id: Optional[str] = None,
     ):
         self._loop = event_loop
         self._deployment_name = deployment_name
+        self._self_node_id = self_node_id
```

Comment:
Nit: can this be evaluated in `PowerOfTwoChoicesReplicaScheduler.__init__` on the spot instead of passed in when creating the router?

Reply:
Gotcha, thanks a lot for the explanation!